turk-one/.idea/jsLibraryMappings.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="JavaScriptLibraryMappings">
+    <file url="PROJECT" libraries="{meteor-packages-auto-import-browser}" />
```

Comment:
These .idea files, I believe, are from Webstorm and are primarily used to keep preferences saved within somebody's own machine. I'm going add a couple of these that are certainly for preferences only (workspace.xml, for example) to the .gitignore so we can ignore these in the future. No worries for now